pkg/util/common.go

```diff
@@ -1324,6 +1349,17 @@ func PatchReplicationControllerForReplicas(clientset *kubernetes.Clientset, old
 	if err != nil {
 		return nil, err
 	}
+
+	for {
```

Comment:
Should have a reasonable timeout rather than loop forever

Reply:
done